src/tune-hyperparameters/src/main/scala/Hyperparams.scala

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import org.apache.spark.ml.classification._
+import org.apache.spark.ml.param.Param
+
+/**
+  * Provides good default hyperparameter ranges and values for sweeping.
+  * Publicly visible to users so they can easily select the parameters for sweeping.
+  */
+object Hyperparams {
+  /**
+    * Defines the default hyperparameter range for logistic regression.
+    * @return The hyperparameter search space for logistic regression.
+    */
+  def logisticRegression(learner: LogisticRegression): Array[(Param[_], Dist[_])] = {
```

Comment:
All of this should be a case match with a single function as interface, or of these functions should have the same name, scala overrides will kick in from there.

Reply:
renamed all functions to "defaultRange" so they would have the same name